git_machete/cmd.py

```diff
@@ -1285,7 +1305,7 @@ def commits_between(earliest_exclusive, latest_inclusive):
 DIVERGED_FROM_AND_NEWER_THAN_REMOTE = 6
 
 
-def get_relation_to_remote_counterpart(b, rb):
+def get_relation_to_remote_counterpart(b: str, rb: str) -> int:
```

Comment:
Actually... is there any easy mypy-supported way to define an enum here?

Reply:
I think it could be handled like this:
https://stackoverflow.com/questions/52624736/type-annotations-for-enum-value

but let's leave it for another PR (since a bunch of renames will be involved)

Reply:
created https://github.com/VirtusLab/git-machete/issues/117